vagrant/Vagrantfile

```diff
@@ -86,8 +86,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         }
       },
       "aet" => {
+        "base_link" => "https://www.kubiczak.pl/aet",
```

Comment:
How about using maven staging repository?
You could easily deploy snapshot version there without having custom links: https://oss.sonatype.org/content/groups/staging/

Reply:
Maven artifacts are something different than "release artifacts" that we use in cookbook.

"Release artifacts":
- zip packages that groups similar AET stuff: bundles, configs, report app.

Maven artifacts:
- are not grouped in any archives.

We eould need adjust AET cookbook to use Maven artifacts only.
Or we could create new Maven artifacts with ZIP packaging for cookbook.
What do you think?